src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
```

Comment:
Consider using a more descriptive name for the function. `getUser` is too generic and doesn't describe what the function does. 🤔